src/__tests__/e2e.test.js

```diff
@@ -453,6 +453,7 @@ describe('e2e', () => {
 
     @track({ page: 'Page' })
     class Page extends React.Component {
+      // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
Ah it would have to go on the line that's giving the error `L460`. You can probably just disable this rule for the entire file, and put this at the top:

```js
/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Reply:
ah. fixed. thanks @tizmagik